addons/website_sale/models/website.py

```diff
@@ -198,7 +198,11 @@ def get_current_pricelist(self):
         return pl
 
     def sale_product_domain(self):
-        return [("sale_ok", "=", True)] + self.get_current_website().website_domain()
+        if self.env.user.has_group('base.group_portal') or self.env.user.has_group('base.group_public'):
```

Comment:
This is not related to the diff in website, right?
It's for regular controller, while the diff in website is for the search behavior, no?
If so, separate both fixes in different commits

Reply:
This part got removed from the PR?
Was it moved in another PR? It was fine to keep it here, just in another commit :)
But let's open another PR now that you removed it, if this is still needed

Reply:
It is not mandatory, I was believing that it could help but not in this case.